irepela/homework_7/retrying.py

```diff
@@ -0,0 +1,29 @@
+"""
+Create decorator, which will retry inner function multiple times until it passes
+"""
+
+
+import random
+
+
+def retry(func):
+
+    def wrapper(*args, **kwargs):
+        while True:
+            try:
+                res = func(*args, **kwargs)
+                break
```

Comment:
in this case you can return at once not to bother with breaking the loop, 
but your variant is also ok